server/athenian/api/internal/features/github/check_run_metrics_accelerated.pyx

```diff
@@ -52,8 +52,13 @@ def calculate_interval_intersections(starts: np.ndarray,
     time_offset = 64 - __builtin_clzl(max_intervals - 1)
```

Comment:
Shall we do the same check here? I can easily image `max_intervals` to be equal to 1...

Reply:
right, done